src/App.js

```diff
@@ -96,13 +101,32 @@ const App = ({ stateCode, correctionsData, jailsData }) => {
     releasesChartData.sourceData,
   ]);
 
+  const { jailsKeyInsightsData, coveredCounty } = generateJailsKeyInsightsData(jailsMetricData);
+
+  const incarcerationRateChartData = generateJailsChartData(
+    jailsMetricData,
+    INCARCERATION_RATE_JAIL,
+    ["Statewide", "US_CO_ARAPAHOE"],
+    ["Statewide", "Arapahoe county"],
```

Comment:
You're planning to keep Arapahoe County hard-coded until #42, I'm guessing?

Reply:
Yes